index.html

```diff
@@ -233,6 +233,58 @@ <h2 class="footer-heading">🟠 Skills</h2>
       </div>
     </div>
   </section>
+  <section class="contact">
+      <h2 class="conclusion-txt">
+      I'm always interested in 
+      hearing about new projects, 
+      so if you'd like to chat please 
+      get in touch.
+    </h2>
+    <form class="form" action="https://formspree.io/html/" method="post">
+      <div class="input-username">
+        <input
+      class="input-1"
+      type="text" value="Arnold"
+      maxlength="30"
```

Comment:
- [ ] Kindly 😊 try to remove the value of the input because you make it static and the user has to remove it to write his info, so please try to make it placeholder like the Figma Design 😃

> Check below to see the difference 😊⤵️
> If you can not see the image clearly just click on it to open a new tab in the browser😊

| My form | Your form| 
| --- | --- |
|  ![Recording 2023-02-28 at 22 08 09](https://user-images.githubusercontent.com/83360636/221966917-3d89102e-00fc-46e3-973b-1ff53c21dd4a.gif)  |    ![Recording 2023-02-28 at 22 06 41](https://user-images.githubusercontent.com/83360636/221966629-2b78704d-d05c-4b83-9389-895c22c9ea1c.gif) |